geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreFactory.scala

```diff
@@ -51,5 +52,6 @@ object CassandraDataStoreParams {
   val CONTACT_POINT = new Param("geomesa.cassandra.contact.point"  , classOf[String], "HOST:PORT to Cassandra",   true)
   val KEYSPACE      = new Param("geomesa.cassandra.keyspace"       , classOf[String], "Cassandra Keyspace", true)
   val NAMESPACE     = new Param("namespace", classOf[URI], "uri to a the namespace", false, null, new KVP(Parameter.LEVEL, "advanced"))
+  val CATALOG       = new Param("catalog", classOf[String])
```

Comment:
let's standardize this as `geomesa.cassandra.catalog.table` - we should probably try to namespace parameters like this going forward.